packages/bugsnag_flutter/lib/src/client.dart

```diff
@@ -769,6 +779,49 @@ class Bugsnag extends BugsnagClient with DelegateClient {
 
     return const <String>{};
   }
+
+  @override
+  networkInstrumentation(data) {
```

Comment:
This implementation really belongs in the `BugsnagClient` class I think, since all client implementations should implement the same network instrumentation logic?